gitmod/matomo.js

```diff
@@ -1,33 +1,72 @@
 addGainsight();
 var checkRequiredElementsExist = setInterval(function () {
-  if (window.gl !== 'undefined' && document.readyState == "complete" && document.querySelectorAll('[data-project]').length) {
-    clearInterval(checkRequiredElementsExist);
-    hideThings();
-    gainsightIdentify();
+    if (window.gl !== 'undefined' && document.readyState == "complete" && document.querySelectorAll('[data-project]').length) {
+      clearInterval(checkRequiredElementsExist);
+      hideThings();
+      gainsightIdentify();
+    }
+  }, 100);
+
+var oldURL = window.location.href;
+setInterval(checkURLchange, 1000);
+
+/**
+ * Check if there is change in URL. If the change observed invoke hideThings();
+ *
+ */ 
+function checkURLchange(){
```

Comment:
The button still shows for me when viewing an individual file. I even tried to re-run the `hideThings` function myself when on the page via the console, but it is not being hidden. Something about `hideThings`  is not properly hiding the li when viewing a file.

https://code.acquia.com/BeverlyHicks/beverly-hicks/-/blob/main/.gitignore?ref_type=heads

![Screenshot 2024-02-27 at 12 45 17 PM](https://github.com/acquia/gitlab-matomo-js/assets/15216074/8bda210d-7abb-4f90-afc1-732c2a427a51)

Reply:
let me verify this